src/man/sssd-ldap.5.xml

```diff
@@ -234,6 +234,13 @@
                                         userPassword (not recommended).
                                     </para>
                                 </listitem>
+                                <listitem>
+                                    <para>
+                                        exop_force - Try Password Modify
+                                        Extended Operation (RFC 3062) even if
+                                        there are no grace logins left.
```

Comment:
Maybe we should say that this is actually expected to fail?

Reply:
I added another sentence.